.snyk

```diff
@@ -0,0 +1,4 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```

Comment:
What does this file do?

Reply:
@raulnegreiros it adds the patches and ignores that were generated by running `snyk wizard`.
When you ignore some vulnerability, it is ignored for 30 days, before being checked again. It can be ignored, for example, if there is no fix for it at the moment.